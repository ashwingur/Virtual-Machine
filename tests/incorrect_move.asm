FUNC LABEL 0
    MOV STK A VAL 5
    PRINT STK A
    MOV VAL 10 STK A
    RET