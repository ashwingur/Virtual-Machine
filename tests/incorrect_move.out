5
Incorrect destination type for MOV.
